backend/README.md

```diff
@@ -1,7 +1,10 @@
 # README
 
 # 🐳 Run (with docker)
-
+- Add `"true"` to the `ENV["DOCKER_ENV"]` variable, by using the following command in your terminal.
```

Comment:
For docker, the environment variables configuration is done by `docker-compose.yml` these instructions are not correct for docker.

What I've suggested for you was to add instructions related to the setup without docker. So people wouldn't suffer as much as you had to setup the project.

Reply:
done!